apps/wing/src/cli.ts

```diff
@@ -3,6 +3,7 @@ import { satisfies } from "compare-versions";
 
 import { optionallyDisplayDisclaimer } from "./analytics/disclaimer";
 import { exportAnalytics } from "./analytics/export";
+import { SNAPSHOTS_HELP } from "./commands/test/snapshots";
```

Comment:
Command-specific imports are typically done dynamically, so putting this at the top here adds a bunch of imports that may never be needed depending on the command ran.

I get the sentiment of wanting to put the help text next to the command but as a quick fix for the benchmark I would say either to inline it in this file or put it as the lone export in another file.

Reply:
OK, done.